src/submit.py

```diff
@@ -43,6 +47,21 @@ async def _check_git_repo(repo_dir: pathlib.Path):
         click.echo(f"Uncommitted changes in {repo_dir}:")
         click.echo(output)
         click.confirm("Are you sure you want to continue?", abort=True)
+        
+    settings_info = settings.get_settings()
+    has_internet_access = bool(settings_info.permissions)
+    
+    if has_internet_access:
+        return await _verify_git_repo_pushed(repo_dir)
+    
+    click.confirm(
+        "Since this task is running on a container with no internet access, "
+        "please copy the repo to your local machine (on your local machine, run `git clone ssh://baseline/home/agent baseline && cd baseline`), and then push the code from your local machine to github (let us know if you don't have the url). Let us know if you need help"
```

Comment:
Need complete, explicit instructions here. The baseliner should NOT be prompted to ask for help, that's a big blocker for them and bottleneck for us.

Reply:
Merging the discussion to here:
https://github.com/poking-agents/headless-human/pull/29#discussion_r1834542053